stencil-library/storybook-pages/intro.stories.mdx

```diff
@@ -135,6 +135,10 @@ This is a complete list of variables as defined in this project's `root.scss`:
   <CSSVars />
 </SummaryElement>
 
+<Markdown>{`
+  **Note:** When customizing the \`--jfi-form-control-box-shadow\` variable, ensure that the \`--jfi-layout-padding\` is not set to \`0\`. This prevents the box-shadow of the last input from being inset into the iframe container, ensuring it remains visible.
```

Comment:
Maybe note that the layout padding needs to be the size of the box shadow, to create room around the inputs so that they don't get cut off